### ISSUE TITLE
Detailed comments.

### ISSUE THREAD

**Issue by ghost** [vahidk/EffectiveTensorflow#27]
The material is great. I can understand how Tensorflow works and go deeper. But there is a lack of some commenting on special arguments. As an example, in the stochastic gradient descent code, there are some parameters for the axis values that I can't understand. It would be great if you provide them too. I know that's a little too much too ask, but for a beginner like me, that's crucial. I've also tried reading the docs of Tensorflow, but they are not good at explaining the axis terms in a simple and understandable way.
Thanks!

**Comment by vahidk** [comment]
Thanks for the feedback. I try to add more details when I got time.
